packages/ui/src/SearchDialog.tsx

```diff
@@ -67,9 +127,9 @@ export function SearchDialog({ tracks }: { tracks: (Track & { problems: Problem[
             <span className="sr-only">Close</span>
           </DialogClose>
         </div>
-        <div className="h-[400px] overflow-y-scroll">
-          {searchTracks.map((track) => (
-            <TrackList key={track.id} track={track} />
+        <div className="h-[400px] overflow-y-scroll" onKeyDown={autoScroll} ref={searchFilterListRef}>
```

Comment:
why the onKeydown here?

Reply:
Added this to check if the next Tracklist is visible in the container. If not will make it visible.

Reply:
> Added this to check if the next Tracklist is visible in the container. If not will make it visible.

Sorry. Haven't cleared the testing things.